content/docs/examples/blog-posts.md

```diff
@@ -0,0 +1,81 @@
++++
+title = "Blog posts"
+description = "The following blog posts provide detailed examples and use cases. Be aware that
+a blog post describes the interfaces at the time of publication of the post.
+Some interfaces are under rapid development and therefore may change frequently."
+weight = 30
++++
+
+{{% blocks/content-item title="The Kubeflow blog"
+  url="https://medium.com/kubeflow" %}}
+Visit the Kubeflow blog to keep up to date with news about the project and to
+learn how to use the latest features.
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="Using Kubeflow to train and serve a PyTorch model in Google Cloud Platform"
+  date="January 23, 2019"
+  url="https://medium.com/kubeflow/end-to-end-kubeflow-tutorial-using-a-pytorch-model-in-google-cloud-platform-10fef557a089" %}}
+This example demonstrates how you can use Kubeflow to train and serve a
+distributed Machine Learning model with PyTorch on a Google Kubernetes Engine
+cluster in Google Cloud Platform (GCP).
+{{% /blocks/content-item %}}
+
+
+{{% blocks/content-item title="Getting started with Kubeflow Pipelines"
+  date="November 16, 2018"
+  url="https://cloud.google.com/blog/products/ai-machine-learning/getting-started-kubeflow-pipelines" %}}
+This article describes how you can tackle ML workflow operations with
+Kubeflow Pipelines, and highlights some examples that you can try
+yourself. The examples revolve around a TensorFlow ‘taxi fare tip prediction’
+model, with data pulled from a public BigQuery dataset of Chicago taxi trips.
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="How to create and deploy a Kubeflow machine learning pipeline"
+  date="November 22 - December 4, 2018"
+  url="https://towardsdatascience.com/how-to-create-and-deploy-a-kubeflow-machine-learning-pipeline-part-1-efea7a4b650f" %}}
+ A series of articles that walk you through the process of taking an existing
+ real-world TensorFlow model and operationalizing the training, evaluation,
+ deployment, and retraining of that model using Kubeflow Pipelines.
+ [Part 1](https://towardsdatascience.com/how-to-create-and-deploy-a-kubeflow-machine-learning-pipeline-part-1-efea7a4b650f)
+ (creating and deploying a pipeline), and
+ [part 2](https://towardsdatascience.com/how-to-deploy-jupyter-notebooks-as-components-of-a-kubeflow-ml-pipeline-part-2-b1df77f4e5b3)
+ (using Jupyter notebooks).
+{{% /blocks/content-item %}}
+{{< /blocks/content-section >}}
```

Comment:
Is the `blocks/content-section` needed? Check in all files.

Reply:
That's a good point. You can remove the `blocks/content-section` markup from all the files, and replace them with headings where necessary.